packages/block-library/src/tag-cloud/edit.js

```diff
@@ -82,6 +89,20 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 					max={ MAX_TAGS }
 					required
 				/>
+				<NumberControl
+					label={ __( 'Smallest Font Size (Pt)' ) }
+					value={ smallestFontSize }
+					onChange={ ( value ) =>
+						setAttributes( { smallestFontSize: value } )
+					}
+				/>
+				<NumberControl
+					label={ __( 'Largeest Font Size (Pt)' ) }
+					value={ largestFontSize }
+					onChange={ ( value ) =>
+						setAttributes( { largestFontSize: value } )
+					}
```

Comment:
We should probably add some defaults, at least for start. Maybe something like the below? (🤔 )
```
min={ 8 }
max={ 60 }
```

Reply:
Sure, agree with you.